tests/mongo/test_mongo.py

```diff
@@ -305,7 +304,7 @@ def test_status_bad_host(mongo_connector):
         ('Host connection', None),
         ('Authenticated', None),
     ]
-    assert 'not known' in status['error']
+    assert len(status['error']) > 0
```

Comment:
Pourquoi avoir changé ce test ?

Reply:
le message d'erreur qui est renvoyé dépends de configurations système, et ne contient pas toujours la string "not known"